src/renderers/shared/fiber/ReactFiberErrorLogger.js

```diff
@@ -33,6 +33,9 @@ function logCapturedError(capturedError : CapturedError) : void {
       ? `React caught an error thrown by ${componentName}.`
       : 'React caught an error thrown by one of your components.';
 
+    const callStack = error.stack;
```

Comment:
On line 29, I think you should use `error.name` instead of `"Error"` because it might be a `TypeError` or something custom.